FRAuthenticator/FRAuthenticator/Manager/AccountMigrationManager.swift

```diff
@@ -0,0 +1,288 @@
+//
```

Comment:
We need to think about if we want to make it public or just add import and export methods on the FRClient. My concern is the easy access to the mechanism secrets the developers would have. This was not is the design, so it's something up to discussion.